test/e2e/performanceprofile/functests/1_performance/performance.go

```diff
@@ -324,6 +325,26 @@ var _ = Describe("[rfe_id:27368][performance]", Ordered, func() {
 				Expect(processesFound).To(BeEmpty(), "The node %s has the following processes on the root cgroup: %v", node.Name, processesFound)
 			}
 		})
+
+		It("[test_id:74767] Should have the ocp-tuned-one-shot.service started and ended before kubelet", func() {
+			// https://issues.redhat.com/browse/OCPBUGS-26401
+			for _, node := range workerRTNodes {
+				By("verify that ocp-tuned-one-shot.service end time is before kubelet.service start time")
+				tunedExitTimeProp, err := systemd.ShowProperty(context.TODO(), "ocp-tuned-one-shot.service", "ExecMainExitTimestampMonotonic", &node)
+				Expect(err).ToNot(HaveOccurred())
+				tunedExitTimeStr := strings.Split(strings.TrimSpace(tunedExitTimeProp), "=")[1]
```

Comment:
please use https://pkg.go.dev/strings#Fields

Reply:
Thank you for the review and comment.
I'm not sure how strings.Fields is better in case of ExecMainExitTimestampMonotonic and below, as these are expected to have one key=value pair in one line. do we need it to guarantee that the test doesn't get flaky in case there could be a text after the property's value? could that happen?

Reply:
it's functionally equivalent but it conveys better the intent of the code

Reply:
In addition the one-liner is probably too fragile, let's unpack the one liner

Reply:
ack, thanks for clarifying, updated